extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -112,7 +112,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				await this.createAndAddBookModel(bookPath, !!isNotebook);
 				let bookViewer = vscode.window.createTreeView(this.viewId, { showCollapseAll: true, treeDataProvider: this });
 				this.currentBook = this.books.find(book => book.bookPath === bookPath);
-				bookViewer.reveal(this.currentBook.bookItems[0], { expand: vscode.TreeItemCollapsibleState.Expanded, focus: true, select: true });
+				if (this._bookViewer.visible) {
```

Comment:
I think you should be able to just use `this.revealActiveDocumentInViewlet` instead and pass in the URI for the book. That keeps the logic for revealing consolidated which is nice

Reply:
Looks like createAndAddBookModel already handles the reveal logic, so this other code is redundant. Pushed up another commit that removes it.